### DISCUSSION TITLE
Disable Local User Login

### DISCUSSION THREAD

**Discussion by LickABrick** [snipe/snipe-it#14638] [Q&A] [answered]
Hi, I am syncing users via LDAP but not all users should be able to login. This is easy to do by just setting the login to disabled.
But I also have setup SAML to let Admins of Snipe-IT login.

Currently these users can either login via SAML or with LDAP. How can I disable local user login completely and only allow login via SAML? I a Github Issue I've seen someone mention there is a .env variable for this but I cannot seem to find which one it is.

**Comment by snipe** [comment, marked as answer]
You can set `REQUIRE_SAML` to `true` in your .env.

https://snipe-it.readme.io/docs/configuration#optional-misc
